package/tests/test_api/test_apps.py

```diff
@@ -62,3 +62,18 @@ def test_reload_endpoint_return_304_when_content_has_not_changed(
         # when the etag has changed, the server will return a 200
         response = client.get("/api/reload", headers={"If-None-Match": "new etag"})
         assert response.status_code == 200
+
+
+class TestFaviconEndpoint:
+    @pytest.fixture
```

Comment:
We don't need this pytest.fixture. We can use client which is defined in conftest.py and just do this directly 

```
 def test_favicon_endpoint(self, client):
        response = client.get("/favicon.ico")
        assert response.status_code == 200
        assert response.headers["content-type"] in [
            "image/x-icon",
            "image/vnd.microsoft.icon",
        ]
```

Look at line 11,12 in the same file. And let me know if you have more questions.